web/app/js/components/Navigation.jsx

```diff
@@ -297,6 +302,13 @@ class NavigationBase extends React.Component {
     this.props.history.push(`/namespaces/${this.state.newNamespace}`);
   }
 
+  handleFilterInputChange = e => {
+    let namespaceFilter = regexFilterString(e.target.value);
```

Comment:
I believe that if you just store the value itself in here, you don't need `namespaceFilterInput` or the `if` statement further down. I assume you're doing this to reduce regex compile time?